src/UI/MessageButtons/WebUrlButton.jsx

```diff
@@ -0,0 +1,26 @@
+import React, {Component} from "react";
+import PropTypes from "prop-types";
+
+export default class WebUrlButton extends Component {
+	handleClick = (e) => {
+		e.preventDefault();
+
+		window.open(this.props.payload, "_blank").focus();
```

Comment:
https://owasp.org/www-project-web-security-testing-guide/latest/4-Web_Application_Security_Testing/11-Client-side_Testing/14-Testing_for_Reverse_Tabnabbing

Dit kan via het 3e attribuut https://developer.mozilla.org/en-US/docs/Web/API/Window/open#noopener

_denk ik overdreven bij CallButton, maar voor de volledigheid daar miss ook neerzetten. Aangezien we niks hebben dat afdwingt dat de call button tel: bevat toch?_

Reply:
Goeie, alleen ondersteunen wij de vulnerable browsers niet https://github.com/parley-messaging/web-library/blob/8fef62cce58fa5f777fd55c41d59d5b67c948253/package.json#L133-L137

Maar kleine moeite om dit toe te voegen.
Done, 4f5c7a1702a84667a60ab286c3594ca67468508e

Ik zag alleen wel dat ze zeggen dat je ook `noreferer` moet gebruiken, of willen wij die in dit geval juist wel aan houden?

> It is recommended to make sure that the rel HTML attribute is set with the noreferrer and noopener keywords for all links.

En in de mdn docs zag ik

> [noreferrer](https://developer.mozilla.org/en-US/docs/Web/API/Window/open#noreferrer)
> If this feature is set, the browser will omit the [Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) header, as well as set noopener to true. See [rel="noreferrer"](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noreferrer) for more information.

Reply:
Done, c96d0a4cbb5bf0061dde3b6ec3899c2df1a3a1ff